### PR TITLE
SEOImages: User Story 1463880, Part 12

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031500050/Report/

File: docs/framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md [hunk 1]
```diff
@@ -72,21 +72,21 @@ Workflow services and clients can participate in transactions.  For a service op
...
   
-     ![Adding a WriteLine activity](../../../../docs/framework/wcf/feature-details/media/addwriteline.JPG "AddWriteLine")  
+     ![Adding a WriteLine activity to the Sequential Service activity(./media/flowing-transactions-into-and-out-of-workflow-services/add-writeline-sequential-service.jpg)  
   
 4.  Drag and drop a <xref:System.ServiceModel.Activities.TransactedReceiveScope> after the <xref:System.Activities.Statements.WriteLine> activity. The <xref:System.ServiceModel.Activities.TransactedReceiveScope> activity can be found in the **Messaging** section of the **Toolbox**. The <xref:System.ServiceModel.Activities.TransactedReceiveScope> activity is composed of two sections **Request** and **Body**. The **Request** section contains the <xref:System.ServiceModel.Activities.Receive> activity. The **Body** section contains the activities to execute within a transaction after a message has been received.  
   
-     ![Adding a TransactedReceiveScope activity](../../../../docs/framework/wcf/feature-details/media/trs.JPG "TRS")  
+     ![Adding a TransactedReceiveScope activity](./media/flowing-transactions-into-and-out-of-workflow-services/transreceivescope-activity.jpg)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md]: Should we use the full name here? 
```suggestion
     ![Adding a TransactedReceiveScope activity](./media/flowing-transactions-into-and-out-of-workflow-services/transactedreceivescope-activity.jpg)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks really good. Just had a question re. one of the file names.

**Comment by damabe** [comment]
@mairaw Yes. Good catch. I've updated the file name and related link. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031700048/Report/.

**Review by mairaw — APPROVED** [review]

**Pull request merged**